alicloud/service_alicloud_rds.go

```diff
@@ -795,6 +795,49 @@ func (s *RdsService) DescribeDbInstanceMonitor(id string) (monitoringPeriod int,
 	return monPeriod, nil
 }
 
+func (s *RdsService) DescribeSQLCollectorPolicy(id string) (*rds.DescribeSQLCollectorPolicyResponse, error) {
+	policy := &rds.DescribeSQLCollectorPolicyResponse{}
+	request := rds.CreateDescribeSQLCollectorPolicyRequest()
+	request.DBInstanceId = id
+	request.RegionId = s.client.RegionId
+	raw, err := s.client.WithRdsClient(func(rdsClient *rds.Client) (interface{}, error) {
+		return rdsClient.DescribeSQLCollectorPolicy(request)
+	})
+	if err != nil {
+		if IsExceptedErrors(err, []string{InvalidDBInstanceIdNotFound, InvalidDBInstanceNameNotFound}) {
+			return policy, WrapErrorf(err, NotFoundMsg, AlibabaCloudSdkGoERROR)
+		}
+		return policy, WrapErrorf(err, DefaultErrorMsg, id, request.GetActionName(), AlibabaCloudSdkGoERROR)
+	}
+	addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+	response, _ := raw.(*rds.DescribeSQLCollectorPolicyResponse)
+
+	// Compatible with api problem of inconsistent input and output
+	if response.SQLCollectorStatus == "Disabled" {
+		response.SQLCollectorStatus = "Disable"
```

Comment:
why

Reply:
> why

Compatible with api problem of inconsistent input and output

Reply:
> why

this api interface is wrong，need compatibility